psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
```

Comment:
Here are two possible ways to handle this particular case that I think are okay.

1) Define `RUN_CCTRANSORT` in the pasture plugin. And then the block here would look like:

```python
try:
    from psi4.driver.pasture import addins
    addint.cctransort_override(ref_wfn)
except ImportError:
     # pasture is not installed
    core.cctransort(ref_wfn)
```
where `addins.cctransort_override()` is going to check the value of `RUN_CCTRANSORT` and decide if it should use `psi4.core.cctransort(ref_wfn)` or `psi4.core.plugin(transqt2_sofile,ref_wfn)` + `psi4.core.plugin(ccsort_sofile,ref_wfn)`

This would be business as usual since pasture would defines the default for `RUN_CCTRANSORT` as true. If pasture is not installed `RUN_CCTRANSORT` is not a valid argument so there is an error if it is used file. If `RUN_CCTRANSORT` is not used and pasture is not installed we end up in the `except` and we use the only thing we have. 


An alternate would be to put the `run_cctransort` option back in and then:

```python

if core.get_global_option('RUN_CCTRANSORT'):
    core.cctransort(ref_wfn)
else:
       try:
           from psi4.driver.pasture import addins
           addins.ccsort_transqt2(ref_wfn)
        except ImportError:
           raise PastureNotFoundError('RUN_CCTRANSORT')
```

and then a message like:

```
RUN_CCTRANSORT = false requires that [ccsort, transqt2] are installed from psi4pasture

 - How to build psi4 with pasture enabled 
 - How to install pasture into an existing psi4 install
```

The exception class could lookup `[ccsort,transqt2]` in a dict somewhere with the key `RUN_CCTRANSORT` that gets expanded when/if other modules move there. 

I like the latter, since it is self documenting, and can be expanded when/if more modules move out to pasture. The pasture requirement could be documented with the option in the manual, and once this is all working we can put a link in the manual to the pasture readme or wherever the install instructions end up.